<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRoundOTF</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>구글 한 줄 번역기</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="kor_input">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>350</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRoundOTF</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px dotted rgb(224, 179, 255);
background-color:rgb(247, 250, 252);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_korInput">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>65</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRoundOTF</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(60, 0, 255);</string>
    </property>
    <property name="text">
     <string>한국어 입력</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_eng">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>180</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRoundOTF</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>영어</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_jap">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>290</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRoundOTF</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>일본어</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_chn">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>405</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRoundOTF</family>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>중국어</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="trans_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>86</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRoundOTF</family>
     </font>
    </property>
    <property name="text">
     <string>번역실행</string>
    </property>
   </widget>
   <widget class="QPushButton" name="init_btn">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>505</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRoundOTF</family>
     </font>
    </property>
    <property name="text">
     <string>초기화</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="eng_output">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>205</y>
      <width>440</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRoundOTF</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove rgb(255, 178, 209);
background-color:rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="jap_output">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>315</y>
      <width>440</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRoundOTF</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove rgb(255, 178, 209);
background-color:rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="chn_output">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>440</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRoundOTF</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px groove rgb(255, 178, 209);
background-color:rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="erase_input_btn">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>118</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>NanumSquareRoundOTF</family>
     </font>
    </property>
    <property name="text">
     <string>지우기</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
